docs/faq.md

```diff
@@ -175,7 +178,12 @@ If it's not sendable, the command will hang for a long time and there will be no
 
 ### General SMTP testing
 
-For general testing of what your SMTP server accepts, **swaks** ([https://jetmore.org/john/code/swaks/](https://jetmore.org/john/code/swaks/)) is a general-purpose tool that can connect using TLS and authenticate with the SMTP server. It requires perl which is not available in Kubernetes nodes of Hubs, but is installed on most *nix machines.
+For general testing of what your SMTP server accepts, **swaks** ([https://jetmore.org/john/code/swaks/](https://jetmore.org/john/code/swaks/)) is a general-purpose tool that can connect using TLS and authenticate with the SMTP server. It requires Perl which is not available in the existing pods of Hubs, but is installed on most *nix machines. 
+
+You can temporarily add a pod to your cluster from which you can run `swaks`:
+```yaml
+kubectl run -it busybox --image=busybox --restart=Never -n hcce
```

Comment:
I guess busybox changed what it bundles because when running this command now `swaks` isn't present.  The `busybox` pod was also left in an error state after exiting and had to be manually deleted.